files/es/web/http/basics_of_http/choosing_between_www_and_non-www_urls/index.md

```diff
@@ -42,7 +42,7 @@ Ejemplo:
 
 El [HTML5 boilerplate project](https://github.com/h5bp/html5-boilerplate) tiene un ejemplo sobre [cómo configurar un servidor Apache para redirigir un dominio a otro](https://github.com/h5bp/html5-boilerplate/blob/7a22a33d4041c479d0962499e853501073811887/.htaccess#L219-L258).
 
-### `Usando <link rel="canonical">`
+### Usando \<link rel="canonical">
```

Comment:
```suggestion
### Usando `<link rel="canonical">`
```

Reply:
Thank you for advice, but the current en-us version uses my style.

https://github.com/mdn/content/blob/3da9465db662eb3a39e3aea413ef4d381637eab3/files/en-us/web/http/basics_of_http/choosing_between_www_and_non-www_urls/index.md?plain=1#L43

Also, the zh version, which also has same change, is already approved.

https://github.com/mdn/translated-content/commit/600fd1967fdeefe767d1edfb1c38d762171d2097#diff-97241c69b852c4183d0d0ddff2c49feb89278322af1c8985d1191eabd971f044R39

How do you think about this? Is there something es specific?

Reply:
Thank you! Maybe @nschonni could tell us if is there some already defined

Reply:
I don't really agree with it, but the current guidelines say not to use code spans in headings https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide#heading_levels

Reply:
I see so, meanwhile yours is correct @mikecat thank you so much ! with this es good to `es`